src/Analyzers/CSharp/CodeFixes/UseCollectionInitializer/CSharpUseCollectionInitializerCodeFixProvider_CollectionInitializer.cs

```diff
@@ -23,7 +23,7 @@ internal partial class CSharpUseCollectionInitializerCodeFixProvider
 {
     private static BaseObjectCreationExpressionSyntax CreateObjectInitializerExpression(
         BaseObjectCreationExpressionSyntax objectCreation,
-        ImmutableArray<Match<StatementSyntax>> matches)
```

Comment:
made non-generic.  we wanted to be able to pass along either a Statement or Expression now, so i just typed this as SyntaxNode.